src/backend/app.js

```diff
@@ -21,10 +22,114 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//app.get("/my-route", (req, res) => {
+//res.send("Hi friend");
+//});
+
+const todayDate = new Date();
+//future meals
+app.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex
+      .select("*")
+      .from("meal")
+      .where("meal_time", ">", todayDate);
+    if (futureMeals.lenght > 0) {
+      res.status(200).json(futureMeals);
+    }
+    res
+      .status(404)
+      .send("Not found");
+  } catch (err) {
+    console.error("Server Error", err);
+    res.status(500).send("Error: Unable to fetch");
+  }
+});
+
+//past meals
+app.get("/past-meals", async (req, res) => {
```

Comment:
maybe the enpoints could be structured differently? for example instead of `/past-meals` you could have `/meals/past` and so on? that way from the `/meals`  with get method you can get all meals and then you can be more specific. I think those are called collections in API design (but again I am a frontender so far from expert :) ) You can read a bit more in this article https://swagger.io/resources/articles/best-practices-in-api-design/